Kentico.Kontent.Management.Tests/CodeSamples/CmApiV2.cs

```diff
@@ -1712,6 +1788,68 @@ public async void PutEnableWebhook()
         Assert.Null(exception);
     }
 
+    // DocSection: cm_api_v2_put_workflow
+    // Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
+    [Fact]
+    public async void PutWorkflow()
+    {
+        var client = _fileSystemFixture.CreateMockClientWithResponse("Workflow.json");
+
+        var response = await client.UpdateWorkflowAsync(Reference.ByCodename("my_workflow"), new WorkflowUpsertModel
```

Comment:
Nitpick: it would be great to let developers know that they can also use Id when updating workflows.

```csharp
var identifier = Reference.ByCodename("my_workflow");
// var identifier = Reference.ById(Guid.Parse("f4b3fc05-e988-4dae-9ac1-a94aba566474"));
```

Reply:
fixed